projects/demo-cypress/src/tests/elastic-sticky.cy.ts

```diff
@@ -36,7 +37,8 @@ describe('ElasticSticky', () => {
 
     beforeEach(() =>
         cy.mount(TestComponent, {
-            imports: [TuiRootModule, TuiScrollbarComponent, TuiElasticStickyModule],
+            imports: [TuiRootComponent, TuiScrollbarComponent, TuiElasticStickyModule],
```

Comment:
Like here, I don't think we need component, but we need plugins. Some other tests are also like that.

Reply:
What do you mean? We have `<tui-root>` inside test component

Reply:
without TuiRootComponent 
<img width="1155" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/6c5024bb-a50c-49fc-8e95-ebfaeef7e367">

Reply:
I don't know what is searching for `tui-root` here, but we do not need it here. We only had it there before to add event plugins. Mostly we do not need `tui-root` for anything, other than portal based components (dropdowns, dialogs, alerts, hints).

Reply:
I can create an issue about `tui-root` in e2e tests and deal with the error

Reply:
Sounds good 👍